ios/Gekidou/Package.resolved

```diff
@@ -1,13 +1,94 @@
 {
   "object": {
     "pins": [
+      {
```

Comment:
Can you revert?  This changes are not necessary.

Reply:
yeah I I revert these changes right before merging. No idea why my system always makes these changes. Any guesses?